### PR TITLE
Add prop renderUrl

### PR DESCRIPTION
**Pull request by LeoIannacone**
Fixes https://github.com/react-component/pagination/issues/61

### CODE REVIEW

File: examples/default.js [hunk 1]
```diff
@@ -14,8 +14,18 @@ class App extends React.Component {
       current: page,
     });
   }
+  renderUrl = (page) => {
+    return `${location.pathname}?page=${page}`;
```
Comment (benjycui) [inline examples/default.js]: -1 for this.

**Comment by benjycui** [comment]
I thinks https://github.com/react-component/pagination/issues/61#issuecomment-283839795 is good enough. We can render `Link` or `a`, and we can event listen events by return `<a onHover={...} />`

**Comment by paranoidjk** [comment]
@LeoIannacone 
Thanks for your contribution, I will close this PR, since IMO `renderUrl` is too narrow. I Agree with `Pagination[pagerContentRender]`,  whould like to open a another PR to help us do it ?

**Comment by LeoIannacone** [comment]
Having a Link (react-router) there is definitely something that I'm interested in, but actually not my priority atm.

Will help anyway as soon as I get some spare time. 

Thanks for the suggestion.

**Pull request closed without merging**